Task_Position_the_Cubes/index.css

```diff
@@ -1,27 +1,65 @@
-
-body {
-  background-color: lightblue;
-}
-p{
-  color: darkblue;
-  background-color: yello;
-}
-h1{
-  color: darkblue;
-  /* background-color: hotpink!important; */
+/*  Start coding below for POSITION THE CUBES TASK*/
+.flexbox-container {
+  width: 300px;
+  height: 300px;
+  margin: center; 
+  background-color:blue;
+  flex-grow:3;
+  position: static;
+ 
 }
-h3{
-  color:darkblue !important;
-  /* background-color: hotpink !important; */
+.flexbox-item-1{
+background: #19c474;
+width: 100PX;
+height: 100PX;
+align-self: center;
+position: relative;
+border: 2px solid white;
+
+} 
+.flexbox-item-2{
+  background: orange;
+  width: 100PX;
+  height: 100PX;
+  align-self: center;
+  margin-left: 66%;
+  margin-top: 32%;
+  border: 2px solid white;
+  
+  
 }
-#secondpara{
-  color: blue;
+.flexbox-item-3{
+background: yellow;
+width: 100px;
+height: 100px;
+margin-bottom: 20%;
+margin-top: -36%;
+margin
+position:relative ;
+border: 2px solid white;
+
 }
-  /* .bgblue{
```

Comment:
There is a missing semicolon at the end of the 'order' property declaration in the 'flexbox-item-5' class.